docs/source/markdown/podman-container-checkpoint.1.md

```diff
@@ -9,6 +9,8 @@ podman\-container\-checkpoint - Checkpoints one or more running containers
 ## DESCRIPTION
 **podman container checkpoint** checkpoints all the processes in one or more *containers*. A *container* can be restored from a checkpoint with **[podman-container-restore](podman-container-restore.1.md)**. The *container IDs* or *names* are used as input.
 
+*IMPORTANT: If the container is using __systemd__ as __entrypoint__ checkpointing the container might not be possible.*
```

Comment:
Can you also mention the `--systemd=always` flag? Forces systemd mode even for containers without systemd as PID 1

Reply:
Running a container with `--systemd=always` does not break checkpointing if systemd is not actually running:
```
podman run -d --systemd=always ubi8-init sh -c "while true; do date; sleep 1; done"
```
This is checkpointable because `sh` is PID 1. Did you mean something else? Not sure I understood it correctly.

Reply:
Ahh, OK. I thought it related to the mounts we were adding for systemd breaking things.

Reply:
systemd uses so many kernel features which most processes are not using and the combination of all of these makes it problematic. bpf, timerfd, fanotify, inotify, raw sockets. Most of these things are supported by CRIU but the combination as seen in systemd makes it problematic. I never really looked at the real problem, because so far using a simpler init system always solved the container use case.